apparmor.d/profiles-m-r/mutt

```diff
@@ -0,0 +1,178 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2023 Zane Zakraisek <zz@eng.utah.edu>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{usr/,}bin/mutt
+profile mutt @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/nameservice>
```

Comment:
Use `nameservice-strict` instead of `nameservice`, and add the required `network` rule in the profile directly.